src/app/api/run/results/route.ts

```diff
@@ -4,16 +4,19 @@ import path from 'path'
 import { UPLOAD_DIR } from '@/app/utils'
 import { parse } from 'csv-parse/sync'
 
-export async function GET() {
+// This disables caching for this endpoint
+export const revalidate = 0
```

Comment:
Next.js n00b question: Probably this is clearly discussed in the docs somewhere I didn't find, but how does it decide when to cache in the first place? Is there a way to tell it to not cache at some point in the hierarchy (e.g. if we knew no API calls should be cached, is there a single place we could specify that)?